bundles/org.openhab.binding.bsblan/src/main/java/org/openhab/binding/bsblan/internal/api/BsbLanApiCaller.java

```diff
@@ -17,10 +17,10 @@
 import java.io.ByteArrayInputStream;
 import java.io.IOException;
 import java.io.InputStream;
-import java.nio.charset.Charset;
 import java.util.Set;
 import java.util.stream.Collectors;
 
+import org.eclipse.californium.elements.util.StandardCharsets;
```

Comment:
Why this strange import? 
You can get UTV 8 standard charger from common Java class.

Reply:
That is where it should have come from.